2-mandatory/1-weather-report.js

```diff
@@ -12,14 +12,18 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let arr = []
```

Comment:
Because the array and the object both are "By Reference", so they could be modified..
```js
const arr = [1,2,3,4,5]
arr.push(2)
console.log(arr) //[ 1, 2, 3, 4, 5, 2 ]

const obj = {}
obj.text='hello'
console.log(obj) //{ text: 'hello' }
```